README.md

```diff
@@ -546,20 +556,82 @@ By default statements are in the *OK* section, but issues that need to
 be addressed should go to the *Problem:* sections (and briefly the
 summary at the top).
 
+# Re-Reviewing an approved package
+
+The MIR team and thereby in extension Ubuntu continuously evolves the MIR
+expectations and guidelines. For example, we add rules based on painful lessons
+learned to avoid the same from happening again. In that regard the demand for
+testing, quality and so on might be best practice now, but have not been
+fulfilled by packages that are in main for a long time.
+
+The proposed approach would be to:
+
+1. We will add (doing so is a TODO) a query that helps us to spot packages in main for the longest time without having a recent update on their MIR bugs.
+1. The MIR team would aim for a constant load of e.g. "1 review per week per MIR member"
+   1. Priority would be on new MIRs, whenever we have spare capacity we would pick up old re-reviews
+   1. On one hand that means if new requests are assigned in a week that we make no progress on the backlog
+   1. But on the other hand this keeps effort of MIR team members somewhat plannable
+1. We'd run a trimmed down check which is a subset of our usual check (see below for details)
+   1. Anything we find for packages already in main becomes part of the owning teams backlog. The decision how hard we push on them is a case by case discussion depending on too many variables to properly pre-define.
+1. As on the initial MIR review we might end up recommending a security (re-)review which goes into their queue at low prio
```

Comment:
please change "we might end up recommending a security (re-)review" to "we might assign a security (re-)review"

I also recommend removing "which goes into their queue at low prio". Priority of a Security MIR is set by the owning team relative to the owning teams other open security MIRs. We try to work on MIRs across all teams evenly, and having owning team set priority helps us know what is most relevant to them. It is exceedingly likely that an owning team will de-prioritize re-reviews though :)